internal/oem/oem.go

```diff
@@ -76,6 +76,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = &[]bool{true}[0]
```

Comment:
Could you use `boolToPtr` in the util package?
https://github.com/coreos/ignition/blob/fe09ef5e14205d022765ebda59390bbee87fd36f/internal/util/util.go#L21